dotcom-rendering/src/layouts/StandardLayout.tsx

```diff
@@ -832,104 +832,95 @@ export const StandardLayout = (props: WebProps | AppProps) => {
 				)}
 
 				{isWeb && (
-					<>
-						<Island priority="feature" defer={{ until: 'visible' }}>
-							<OnwardsUpper
-								ajaxUrl={article.config.ajaxUrl}
-								hasRelated={article.hasRelated}
-								hasStoryPackage={article.hasStoryPackage}
-								isAdFreeUser={article.isAdFreeUser}
-								pageId={article.pageId}
-								isPaidContent={!!article.config.isPaidContent}
-								showRelatedContent={
-									article.config.showRelatedContent
-								}
-								keywordIds={article.config.keywordIds}
-								contentType={article.contentType}
-								tags={article.tags}
-								format={format}
-								pillar={format.theme}
-								editionId={article.editionId}
-								shortUrlId={article.config.shortUrlId}
-								discussionApiUrl={
-									article.config.discussionApiUrl
-								}
-							/>
-						</Island>
+					<Island priority="feature" defer={{ until: 'visible' }}>
+						<OnwardsUpper
+							ajaxUrl={article.config.ajaxUrl}
+							hasRelated={article.hasRelated}
+							hasStoryPackage={article.hasStoryPackage}
+							isAdFreeUser={article.isAdFreeUser}
+							pageId={article.pageId}
+							isPaidContent={!!article.config.isPaidContent}
+							showRelatedContent={
+								article.config.showRelatedContent
+							}
+							keywordIds={article.config.keywordIds}
+							contentType={article.contentType}
+							tags={article.tags}
+							format={format}
+							pillar={format.theme}
+							editionId={article.editionId}
+							shortUrlId={article.config.shortUrlId}
+							discussionApiUrl={article.config.discussionApiUrl}
+						/>
+					</Island>
+				)}
 
-						{!isPaidContent && showComments && (
-							<Section
-								fullWidth={true}
-								sectionId="comments"
-								data-print-layout="hide"
-								element="section"
+				{!isPaidContent && showComments && (
```

Comment:
So if an article is commentable but paid content, we’ll show the comments in the meta, but not display the discussion layout?

I’m wondering if ArticleMeta’s prop should be renamed from `isCommentable` to `showComments` to make this clearer what we mean by these props?